recursio-client/Level/CapturePoint.gd

```diff
@@ -1,14 +1,14 @@
 extends Spatial
 class_name CapturePoint
 
-onready var neutral_color = Color(Constants.get_value("colors", "neutral"))
+onready var neutral_color_name = "neutral"
 # TODO: better use main schema for all coloring?
-onready var player_color = Color(Constants.get_value("colors", "player_main"))
-onready var enemy_color = Color(Constants.get_value("colors", "enemy_main"))
-onready var player_in_capture_color = Color(Constants.get_value("colors", "player_ghost_primary_accent"))
-onready var player_captured_color = Color(Constants.get_value("colors", "player_ghost_main"))
-onready var enemy_in_capture_color = Color(Constants.get_value("colors", "enemy_ghost_primary_accent"))
-onready var enemy_captured_color = Color(Constants.get_value("colors", "enemy_ghost_main"))
+onready var player_color_name = "player_main"
```

Comment:
the `onready` could be removed from these since they're just strings now

Reply:
Fixed